tests/integration/generated/test_debug3_chrysalis.cfg

```diff
@@ -0,0 +1,62 @@
+# Use this cfg to test a pull request before merging.
+# 1. Copy over the contents of template_complete_run.cfg.
+# 2. Delete parts that aren't needed. `output` and `www` should include `zppy_test_debug` rather than `zppy_test_complete_run`.
+# 3. Iteratively debug/develop with this cfg.
+# 4. Once it's working as expected, copy important changes to template_complete_run.cfg.
+
+[default]
+case = v2.LR.BGC-LNDATM.CONTRL
+constraint = ""
+dry_run = "False"
+environment_commands = ""
+#input = "/lcrc/group/e3sm/ac.forsyth2//E3SMv2/v2.LR.historical_0201"
+input = "/lcrc/group/e3sm/ac.sfeng1/E3SM_Simulations/20221127.v2.LR.BGC-LNDATM.CONTRL.ne30pg2_r05_EC30to60E2r2.chrysalis"
+input_subdir = archive/atm/hist
+mapping_file = "map_r05_to_cmip6_180x360_aave.20231110.nc"
+# To run this test, edit `output` and `www` in this file, along with `actual_images_dir` in test_complete_run.py
+output = "/lcrc/group/e3sm/ac.forsyth2/zppy_test_debug3_output/pr-400v42/20221127.v2.LR.BGC-LNDATM.CONTRL.ne30pg2_r05_EC30to60E2r2.chrysalis"
```

Comment:
I used `debug3` to test with data from @BunnyVon's simulation `20221127.v2.LR.BGC-LNDATM.CONTRL.ne30pg2_r05_EC30to60E2r2.chrysalis`. I ran what plots I could -- 6 of the original 8 plots could be generated (the two requiring `RESTOM` and/or `RESSURF` could not be plotted), 3 `atm` plots, and 4 `lnd` plots. Results are available [here](https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.forsyth2/zppy_test_debug3_www/pr-400v42/v2.LR.BGC-LNDATM.CONTRL/global_time_series/global_time_series_1870-1880_results/).

Error 2) the `lnd` plots still appear to be horizontal outside of 1874-1876. I still need to debug that. I believe there were comments earlier on this PR that could help with that.

Question 1) The PDF of `original` plots shows only 6 plots -- 3 of which are the blank ocean plots. 2 plots are missing entirely because I left them off the plot list (those are the ones I mentioned in the first paragraph). The 3 blank ocean plots appear because that is the original behavior -- the ocean plots stay on the page but just end up empty if the data isn't available. Do we want that to continue to be the expected behavior?

Reply:
> I fixed the error in `ncclimo` that caused problems with the regional average timeseries generation for ELM files.

Indeed that should hopefully take care of Error 2.

Reply:
this simulation is an F compset simulation; ocean is not active and used a data model. so you don't see any relevant results or plots. If zppy will generate separated pdfs for plots, I think it makes sense to put atm, land, in the different pdfs. Or, make the option `atmosphere only` atmosphere only.

Reply:
I now realize Question (1) is irrelevant. The plots show up blank because they were still specified in `plots_original`. I guess a better question would be if we want to a) raise an Exception because we don't have the data to plot or b) keep the behavior of just printing empty ocn plots.